test/Makefile

```diff
@@ -0,0 +1,64 @@
+LIBDIR=../lib
+GTEST_MODULE=$(LIBDIR)/googletest
+GTEST_DIR=$(LIBDIR)/googletest/googletest
+GTEST_SOURCEDIR=$(GTEST_DIR)/src
+GTEST_HEADERS=$(GTEST_DIR)/include/gtest/*.h \
+              $(GTEST_DIR)/include/gtest/internal/*.h
+GTEST_SRCS_=$(GTEST_SOURCEDIR)/*.cc $(GTEST_SOURCEDIR)/*.h $(GTEST_HEADERS)
+
+CPP_NETLIB_DIR=$(LIBDIR)/cpp-netlib
+YAML_CPP_DIR=$(LIBDIR)/yaml-cpp
+
+CPPFLAGS+=-isystem $(GTEST_DIR)/include
+CXXFLAGS=-std=c++11 -g -pthread
+
+# Where to find user code.
+SRC_DIR=../src
+SRCS=\
+     $(SRC_DIR)/format.cc
```

Comment:
We should probably make this depend on `../src/format.o` and invoke make recursively to build it.

Reply:
Does the following line match what you want? (L58)
$(OBJS): build-source